third_party/xla_client/mhlo_helper.cc

```diff
@@ -0,0 +1,19 @@
+#include "third_party/xla_client/mhlo_helper.h"
+#include "tensorflow/compiler/xla/translate/hlo_to_mhlo/hlo_to_mlir_hlo.h"
+#include "tensorflow/compiler/xla/translate/mhlo_to_hlo/mlir_hlo_to_hlo.h"
+
+namespace xla {
+
+void hlo_mhlo_hlo_roundtrip_helper(HloModuleProto* proto) {
+  mlir::MLIRContext context;
+  mlir::ModuleOp mlir_module = mlir::ModuleOp::create(mlir::UnknownLoc::get(&context));
+  auto status = ConvertHloToMlirHlo(mlir_module, proto, /*import_all_computations=*/false);
```

Comment:
```
  if (!status.ok()) {
    // Do some error handling / logging 
  }
```

Here and below, will help track where issues are happening.

Reply:
Thanks for the comment. Yes, will add more logging when debugging the tests failure.